.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 matrix:
   include:
-    - php: 5.5
+    - php: 5.6
```

Comment:
i don't think we drop 5.5 for 1.3 release. but we should discuss this in the related issue #2746